Referenced by issue #100363, Crash after running api command bpy.ops.object.modifier_apply()
